File: Big-DicTaphone/Models/Stakeholder.swift
import Foundation

struct Stakeholder: Codable, Identifiable, Equatable {
    let id: UUID
    var name: String
    var email: String
    var role: String?
    
    init(id: UUID = UUID(), name: String, email: String, role: String? = nil) {
        self.id = id
        self.name = name
        self.email = email
        self.role = role
    }
}

File: android/app/src/main/java/com/bigdictaphone/app/data/Stakeholder.kt
package com.bigdictaphone.app.data

import kotlinx.serialization.Serializable
import java.util.UUID

/**
 * Represents a stakeholder who receives management updates
 */
@Serializable
data class Stakeholder(
    val id: String = UUID.randomUUID().toString(),
    val name: String,
    val email: String,
    val role: String? = null  // e.g., "Manager", "Team Lead", "VP"
)
